src/lib.rs

```diff
@@ -230,17 +230,19 @@ macro_rules! vtc {
 macro_rules! boilerplate {
     () => {
         #[allow(
-            non_upper_case_globals,
+            dead_code,
             non_camel_case_types,
             non_snake_case,
+            non_upper_case_globals,
```

Comment:
shouldn't we be removing those?

Reply:
I simply sorted it - we should remove some/most of these, just not yet

Reply:
but `unused_mut` or is it the pre-coffee haze hindering me? Anyway, ok with me

Reply:
I moved `unused_mut` from inside the python-generated script - no point in having some allows generated for every method, while others are here.  Once proc-macro lands, it will become much simpler to fix these